tests/Frontend/views/settings/servers/Index.spec.js

```diff
@@ -0,0 +1,762 @@
+import Index from '../../../../../resources/js/views/settings/servers/Index';
+import { createLocalVue, mount } from '@vue/test-utils';
+import PermissionService from '../../../../../resources/js/services/PermissionService';
+import moxios from 'moxios';
+import BootstrapVue, {
+  IconsPlugin,
+  BTr,
+  BTbody,
+  BButton,
+  BModal,
+  BButtonClose,
+  BFormInput
+} from 'bootstrap-vue';
+import sinon from 'sinon';
+import Base from '../../../../../resources/js/api/base';
+import Vuex from 'vuex';
+
+const localVue = createLocalVue();
+localVue.use(BootstrapVue);
+localVue.use(IconsPlugin);
+
+const defaultResponse = {
+  data: [
+    {
+      id: 1,
+      description: 'Server 01',
+      strength: 1,
+      status: 1,
+      participant_count: 10,
+      listener_count: 5,
+      voice_participant_count: 5,
+      video_count: 5,
+      meeting_count: 2,
+      model_name: 'Server',
+      updated_at: '2020-12-21T13:43:21.000000Z'
+    },
+    {
+      id: 2,
+      description: 'Server 02',
+      strength: 1,
+      status: 1,
+      participant_count: 50,
+      listener_count: 25,
+      voice_participant_count: 30,
+      video_count: 5,
+      meeting_count: 10,
+      model_name: 'Server',
+      updated_at: '2020-12-21T13:43:21.000000Z'
+    }
+  ],
+  links: {
+    first: 'http://localhost/api/v1/servers?page=1',
+    last: 'http://localhost/api/v1/servers?page=1',
+    prev: null,
+    next: null
+  },
+  meta: {
+    current_page: 1,
+    from: 1,
+    last_page: 1,
+    path: 'http://localhost/api/v1/servers',
+    per_page: 15,
+    to: 2,
+    total: 2
+  }
+};
+
+const createContainer = (tag = 'div') => {
+  const container = document.createElement(tag);
+  document.body.appendChild(container);
+  return container;
+};
+
+const store = new Vuex.Store({
+  modules: {
+    session: {
+      namespaced: true,
+      getters: {
+        settings: () => (setting) => setting === 'pagination_page_size' ? 5 : null
```

Comment:
This does not make sense since in line 62 the per_page is 15.

Reply:
True, will change it